piksi_multi_rtk_ros/src/piksi_multi_rtk_ros/piksi_multi.py

```diff
@@ -161,6 +163,13 @@ def __init__(self):
 
         # Other parameters.
         self.publish_raw_imu_and_mag = rospy.get_param('~publish_raw_imu_and_mag', False)
+        # Publish IMU
+        self.acc_scale = 8 * 9.81 / 32768
```

Comment:
is `32768` always stable or would you rather use a constant here?

Reply:
Made the fixed scaling constants constants.